dnf-plugins-core.spec

```diff
@@ -74,18 +102,168 @@ BuildRequires:  python3-sphinx
+Provides:       dnf-plugins-extras-leaves = %{epoch}:%{version}-%{release}
+Conflicts:      dnf-plugins-extras-common-data < %{dnf_plugins_extra}
+Conflicts:      python2-dnf-plugin-leaves < %{epoch}:%{version}-%{release}
+Obsoletes:      python3-dnf-plugins-extras-leaves < %{dnf_plugins_extra}
+
+%description -n python3-dnf-plugin-leaves
+Leaves Plugin for DNF, Python 3 version. List all installed packages
+not required by any other installed package.
+
+%package -n python2-dnf-plugin-local
+Summary:        Local Plugin for DNF
+Requires:       %{_bindir}/createrepo_c
+Requires:       python2-%{name} = %{epoch}:%{version}-%{release}
+Provides:       dnf-plugin-local =  %{epoch}:%{version}-%{release}
+Provides:       python2-dnf-plugins-extras-local = %{epoch}:%{version}-%{release}
+Provides:       dnf-command(local)
+Provides:       dnf-plugins-extras-local = %{epoch}:%{version}-%{release}
+Conflicts:      dnf-plugins-extras-common-data < %{dnf_plugins_extra}
+Conflicts:      python3-dnf-plugin-local < %{epoch}:%{version}-%{release}
+Obsoletes:      python2-dnf-plugins-extras-local < %{dnf_plugins_extra}
+
+%description -n python2-dnf-plugin-local
+Local Plugin for DNF, Python 2 version. Automatically copy all downloaded packages to a
+repository on the local filesystem and generating repo metadata.
+
+%package -n python3-dnf-plugin-local
+Summary:        Local Plugin for DNF
+Requires:       %{_bindir}/createrepo_c
+Requires:       python3-%{name} = %{epoch}:%{version}-%{release}
+Provides:       dnf-plugin-local =  %{epoch}:%{version}-%{release}
+Provides:       python3-dnf-plugins-extras-local = %{epoch}:%{version}-%{release}
+Provides:       dnf-command(local)
+Provides:       dnf-plugins-extras-local = %{epoch}:%{version}-%{release}
+Conflicts:      dnf-plugins-extras-common-data < %{dnf_plugins_extra}
+Conflicts:      python2-dnf-plugin-local < %{epoch}:%{version}-%{release}
+Obsoletes:      python3-dnf-plugins-extras-local < %{dnf_plugins_extra}
+
+%description -n python3-dnf-plugin-local
+Local Plugin for DNF, Python 3 version. Automatically copy all downloaded
+packages to a repository on the local filesystem and generating repo metadata.
+
+%package -n python2-dnf-plugin-migrate
+Summary:        Migrate Plugin for DNF
+Requires:       python2-%{name} = %{epoch}:%{version}-%{release}
+Requires:       yum
+Provides:       dnf-plugin-migrate = %{epoch}:%{version}-%{release}
+Provides:       python2-dnf-plugins-extras-migrate = %{epoch}:%{version}-%{release}
+Provides:       dnf-command(migrate)
+Provides:       dnf-plugins-extras-migrate = %{epoch}:%{version}-%{release}
+Conflicts:      dnf-plugins-extras-common-data < %{dnf_plugins_extra}
+Obsoletes:      python2-dnf-plugins-extras-migrate < %{dnf_plugins_extra}
+
+%description -n python2-dnf-plugin-migrate
+Migrate Plugin for DNF, Python 2 version. Migrates history, group and yumdb data from yum to dnf.
+
+%package -n python2-dnf-plugin-show-leaves
```

Comment:
I don't care what the current leaves plugin ends up being called, but I would like to be able to install only the current leaves plugin and not have to run it through the show-leaves plugin.